qa/tasks/vstart_runner.py

```diff
@@ -719,28 +724,23 @@ def list_connections():
         pre_mount_conns = list_connections()
         log.info("Pre-mount connections: {0}".format(pre_mount_conns))
 
-        prefix = [os.path.join(BIN_PREFIX, "ceph-fuse")]
+        prefix = ['sudo', 'nsenter',
+                  '--net=/var/run/netns/{0}'.format(self.netns_name),
+                  os.path.join(BIN_PREFIX, "ceph-fuse")]
```

Comment:
An acceptable non-`sudo` fallback would be to `SIGKILL` the `ceph-fuse` mount as we did before the network bridge code was added. Let's do that instead?

Reply:
I'll try that out.

Reply:
On a second look, I am not sure what changes are you implying. Can you please elaborate?

Reply:
And, (following up on my above comment), I think we should make `nsenter` conditional on some `vstart_runner.py` option that does network namespace setup. Otherwise, as I suggested earlier, use `SIGKILL` to kill mounts rather than shutting down its network namespace.

Reply:
I've made the change, please take a look.